10_PredictionExamples.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>I'd use more samples here (~1,000) since the model is so simple.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>